examples/simple_fleet_app_operator_permissions/versions.tf

```diff
@@ -20,11 +20,11 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 4.81.0"
+      version = ">= 6.39.0"
```

Comment:
Please update the PR title to capture this breaking change: `feat(TPG>=6.39)!: Fleet app operator permissions custom roles`